GTG/gtk/browser/cell_renderer_tags.py

```diff
@@ -42,24 +43,43 @@ class CellRendererTags(Gtk.CellRenderer):
     # Private methods
     def __roundedrec(self, context, x, y, w, h, r=10):
         "Draw a rounded rectangle"
-        #   A****BQ
-        #  H      C
-        #  *      *
-        #  G      D
-        #   F****E
-
-        context.move_to(x + r, y)                          # Move to A
-        context.line_to(
-            x + w - r, y)                        # Straight line to B
-        # Curve to C, Control points are both at Q
-        context.curve_to(x + w, y, x + w, y, x + w, y + r)
-        context.line_to(x + w, y + h - r)                      # Move to D
+        #   A  *  BQ
+        #  H       C
+        #  *       *
+        #  G       D
+        #   F  *  E
+
+        context.move_to(
+            x + r, y
+        )  # Move to A
```

Comment:
I think you wanted to format like the following `curve_to` calls, but it doesn't have many parameters and maybe could easily led to make one think it is a `curve_to` call. It's not too bad tho.

Reply:
I agree. I've made the changes and pushed them.